cmd/root.go

```diff
@@ -102,6 +107,34 @@ func logLevel() error {
 	return nil
 }
 
+func setupSentryHook() error {
+	sentryEnv := viper.GetString("sentry.env")
+	sentryDSN := viper.GetString("sentry.dsn")
+
+	sentryErr := sentry.Init(sentry.ClientOptions{
+		Dsn:         sentryDSN,
+		Environment: sentryEnv,
+		Debug:       true,
+	})
+	if sentryErr != nil {
+		return fmt.Errorf("error initializing Sentry: %w", sentryErr)
+	}
+
+	sentryHook, hookErr := logrus_sentry.NewSentryHook(sentryDSN, []logrus.Level{
+		logrus.ErrorLevel,
+		logrus.FatalLevel,
+		logrus.PanicLevel,
```

Comment:
One interesting thing here is that I'm not sure we log around all of our `panic`s - assuming we may need to go add that to make this effective?

Reply:
Yes, I suspect so, or handle them [separately](https://docs.sentry.io/platforms/go/panics/).

Reply:
(I'm totally in favor of merging this and doing that in a separate pass, though)

Reply:
Right now I'm leaning toward handing this in a separate story, because it seems like a lot of the built in approaches won't play nice with a `panic` emanating from arbitrary goroutines: https://github.com/getsentry/sentry-go/issues/115#issuecomment-564032753

That said I could totally be reading something wrong and am on board to revisit within this PR.

Sidebar is that I don't think we have that many `panic` statements and could probably replace all of them without too much effort.